packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -378,29 +421,31 @@ class _SnackBarState extends State<SnackBar> {
     Widget snackBar = SafeArea(
       top: false,
       bottom: !isFloatingSnackBar,
```

Comment:
We'll add a SafeArea here that's not used if `isFloatingSnackBar` is true. You could break it out to a conditional below, not sure if its better?

Reply:
Good call, done